sdk/examples/wallet/03_get_funds.rs

```diff
@@ -29,6 +29,9 @@ async fn main() -> Result<()> {
     let faucet_response =
         request_funds_from_faucet(&env::var("FAUCET_URL").unwrap(), &address[0].address().to_bech32()).await?;
 
+    // Ensure the account is synced after receiving funds.
+    account.sync(None).await?;
+
     println!("{faucet_response}");
```

Comment:
The faucet also needs some time to send a transaction, so maybe something like this would be better
```suggestion
    println!("{faucet_response}");
    
    // Wait a few seconds for the faucet transaction to get processed
    tokio::time::sleep(std::time::Duration::from_secs(15)).await;

    // Ensure the account is synced after receiving funds.
    account.sync(None).await?;
```

Reply:
Oof, is there really not a better way than that?

Reply:
Not really, in the tests we do this https://github.com/iotaledger/iota-sdk/blob/f5d4d22ee792300b3664c6a17d42a8916c09973b/sdk/tests/wallet/common/mod.rs#L60-L69
The faucet API doesn't return any information about when the transaction will be send and even then we don't know when it will be confirmed